src/xdist/remote.py

```diff
@@ -146,30 +174,25 @@ def handle_command(
             self.steal(kwargs["indices"])
 
     def steal(self, indices: Sequence[int]) -> None:
```

Comment:
Can we add a docstring here? I think it is important to mention that it will steal all `indices` given if it *can* steal all tests and they are all still pending, or none at all.